packages/jest-cli/src/__tests__/TestRunner-test.js

```diff
@@ -38,6 +38,115 @@ test('.addReporter() .removeReporter()', () => {
+
+// Checking for the method _addCustomReporters
+// Custom reporters used here are the reporters within the package
+// No extra reporters are included to be used
+describe('_addCustomReporters', () => {
+  // Paths for the given reporters
+  const SUMMARY_REPORTER_PATH = './reporters/SummaryReporter.js';
+  const VERBOSE_REPORTER_PATH = './reporters/VerboseReporter.js';
+  const DEFAULT_REPORTER_PATH = './reporters/DefaultReporter.js';
+
+  // Requiring constructors of the given reporters
+  // to check against the reporters added
+  const summaryReporter = require('.' + SUMMARY_REPORTER_PATH);
+  const verboseReporter = require('.' + VERBOSE_REPORTER_PATH);
+  const defaultReporter = require('.' + DEFAULT_REPORTER_PATH);
+
+  let runner;
+
+  beforeEach(() => {
+    runner = new TestRunner({}, {noDefaultReporters: true}, {});
+  });
+
+  test('adds reporter using 2D Array format', () => {
+    const reporters = [
+      [SUMMARY_REPORTER_PATH, {}],
+    ];
+
+    expect(runner._dispatcher._reporters).toHaveLength(0);
+    expect(runner._dispatcher._reporters[0]).toBeUndefined();
+
+    runner._addCustomReporters(reporters);
+
+    expect(runner._dispatcher._reporters).toHaveLength(1);
+    expect(runner._dispatcher._reporters.pop()).toBeInstanceOf(summaryReporter);
+  });
+
+  test('adds reporter using 2D syntax with no configuration object', () => {
+    const reporters = [
+      [SUMMARY_REPORTER_PATH],
+    ];
+
+    runner._addCustomReporters(reporters);
+
+    expect(runner._dispatcher._reporters).toHaveLength(1);
+    expect(runner._dispatcher._reporters.pop()).toBeInstanceOf(SummaryReporter);
+  });
+
+  test('adds reporter using string syntax (no custom configuration)', () => {
+    const reporters = [
+      SUMMARY_REPORTER_PATH,
+    ];
+
+    runner._addCustomReporters(reporters);
+
+    expect(runner._dispatcher._reporters).toHaveLength(1);
+    expect(runner._dispatcher._reporters.pop()).toBeInstanceOf(summaryReporter);
+  });
+
+  test('adds two reporters with variable format', () => {
+    const reporters = [
+      VERBOSE_REPORTER_PATH,
+      [DEFAULT_REPORTER_PATH, {}],
+    ];
+    runner._addCustomReporters(reporters);
+
+    expect(runner._dispatcher._reporters).toHaveLength(2);
+
+    expect(runner._dispatcher._reporters[0]).toBeInstanceOf(verboseReporter);
+    expect(runner._dispatcher._reporters[1]).toBeInstanceOf(defaultReporter);
+  });
+
+  test('throws on invalid file path', () => {
+    const reporters = [
+      ['ohthisisnotgoingtobearealpath.sadfslj', {}],
+    ];
+
+    const addInvalidReporters = () => {
+      runner._addCustomReporters(reporters);    
+    };
+
+    expect(addInvalidReporters).toThrow();
+    expect(addInvalidReporters).toThrow(/Cannot find module/);
+    expect(runner._dispatcher._reporters).toHaveLength(0);
+  });
+
+  test('throws on invalid argument to reporter', () => {
+    expect(() => {
+      runner._addCustomReporters('This should be an array obviously');
+    }).toThrow();
```

Comment:
you should avoid using `toThrow`, because you can accidentally make a typo in your code and it'll throw with `unexpected token` or something, but the test will still be green because it still throws an error